x-pack/plugins/cloud/get_cloud_usage_collector.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Server } from 'hapi';
+import { KIBANA_CLOUD_STATS_TYPE } from './constants';
+
+export interface UsageStats {
+  isCloudEnabled: boolean;
+}
+
+export interface KibanaHapiServer extends Server {
+  usage: {
+    collectorSet: {
+      makeUsageCollector: any;
+    };
+  };
+}
+
+export function createCollectorFetch(server: any) {
+  return async function fetchUsageStats(): Promise<UsageStats> {
+    const { id } = server.config().get(`xpack.cloud`);
+
+    return {
+      isCloudEnabled: !!id,
```

Comment:
any reason to not check that the type is a string?

Reply:
Both should work for our case.

Checking for 'falesy' ids might protect us from unpredicted scenarios (empty string ids etc). More importantly; Checks for `isCloudEnabled` in other places in our code use `!!id` so this also ensures consistency.

Reply:
++ for consistency